wasmCloud/wasmCloud
# feat(example): use published tools for secrets

## Feature or Problem
This PR removes the `build.sh` and `run.sh` scripts from the secrets example in favor of using the secrets-nats-kv CLI and the (currently unreleased) latest wash, wadm, and wasmCloud versions.

~This PR is a draft while I can't put wadm & wasmcloud in the docker compose, and suggest using a beta version of wash. Once those are ready this is good to go.~

I decided to just make this work with `wash up`, and it will work as long as you have `wash` installed from https://github.com/wasmCloud/wasmCloud/pull/2615

## Related Issues


## Release Information


## Consumer Impact


## Testing


### Unit Test(s)


### Acceptance or Integration


### Manual Verification
I did step through the steps in the README and they worked great! If you want to try it out, ensure you're running wash, wasmCloud, and wadm from main. A followup PR will update wash to use the latest wadm / wasmcloud so this can be easier.
